src/lib/InstantSearch.ts

```diff
@@ -151,6 +151,7 @@ class InstantSearch<
   public mainHelper: AlgoliaSearchHelper | null;
   public mainIndex: IndexWidget;
   public started: boolean;
+  public disposed: boolean;
```

Comment:
I'd like to have `public state: 'none' | 'started' | 'disposed'`, but I'm not sure how much `started` being used. At least, it's not documented https://www.algolia.com/doc/api-reference/widgets/instantsearch/js/.

Reply:
started is used now, by Vue InstantSearch for example, but could we not introduce `state` instead of `disposed` now? (we can still set both state and started and deprecate `started`)

Reply:
That sounds more future-proof.

Reply:
edit: 'none' → 'initial'